charts/nexus3/templates/configmap-scripts.yaml

```
@@ -20,7 +20,7 @@ data:
     set -euo pipefail
 
 {{- range .Values.plugins }}
-    curl -sSLo /deploy/{{ .name }}.kar {{ .url }}
+    curl -fSLo /deploy/{{ .name }}.kar {{ .url }}
```

```suggestion
    curl -fLo /deploy/{{ .name }}.kar {{ .url }}
```
Unless you need the `-S` when using `-f` we don't need it?